### DISCUSSION TITLE
对不起，我是笨比，教教偶怎么用::>_<::

### DISCUSSION THREAD

**Discussion by ahehef** [LmeSzinc/AzurLaneAutoScript#3013] [Q&A]
打开ALSA出现   Please unzip ALAS installer first  要怎么办？
那个yaml文件用记事本打开改了一下不知道有没有用
可能我只适合用操作录制/(ㄒoㄒ)/~~

**Comment by lyhggf** [comment]
这哪里是笨比，哪来的SB？

**Comment by Scp-096-r** [comment]
我没理解错我话......？它是要你解压安装程序？

**Reply by ahehef** [reply]
是的，但我不知道怎么操作

**Reply by Scp-096-r** [reply]
最简单的只要解压了之后再打开文件夹，直接打开Alas.exe就行了。
在首页（也就是Code页）的右边，有一个“Releases”，点进Easy Install ，比较小，仔细看看。（在那一堆头像的上面一些）；
下载 AlasApp_0.4.4_fullcn.7z。一定要是这个，别下其他的。
下载完后解压，解压完进去直接打开Alas.exe。
（总不能说不会解压吧）
算了，我直接贴出来：https://github.com/LmeSzinc/AzurLaneAutoScript/releases/tag/v2023.08.20
里面有中英双文教程，解压即用。

**Reply by ahehef** [reply]
好的，已经解决了（找到解压后的文件了）
